src/engraving/tests/utils/scorerw.cpp

```diff
@@ -77,6 +77,15 @@ MasterScore* ScoreRW::readScore(const String& name, bool isAbsolutePath, ImportF
         }
     }
 
+    // While reading the score, some elements might use `score->repeatList()` (which is incorrect
+    // anyway, because the repeatList will be incomplete because the score is incomplete, but some
+    // elements still do it).
+    // `score->repeatList()` calls `_repeatList->update()`; the repeat list then thinks that it is
+    // up-to-date from that point. But we weren't finished reading the score, so the score will still
+    // change. We need to tell the repeat list about that, so that it will be updated next time
+    // someone uses it.
+    score->setPlaylistDirty();
```

Comment:
This change is not strictly necessary anymore since the changes in spanner.cpp/h; I'd still like to have it here, just in case something else happens to use the repeat list while reading the score, and because it is in EngravingPRoject::doSetupMasterScore too.